### PR TITLE
Fix va-modal for Cypress

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Chromatic

https://1113-modal-cypress--60f9b557105290003b387cd5.chromatic.com

## Description
Part of https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1113

## Acceptance criteria
- [x] Cypress tests utilizing va-modal pass 

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by andresriveratoro** [comment]
I believed I fixed the previous error, but now I'm getting a new one. 🙃 
Screen Shot 2022-08-23 at 17 23 07

**Comment by andresriveratoro** [comment]
The current changes pass the Cypress tests 🎉 
The error above was due to an incorrect usage of the web component. Removing `role="alertdialog"` gets rid of the error. Our modal components sets the correct role for a given `status`.
Screen Shot 2022-08-23 at 17 34 03

**Review by bkjohnson — COMMENTED** [review]
Do these changes mess up the OMB Info component at all?

File: packages/web-components/src/components/va-modal/va-modal.tsx [hunk 1]
```diff
@@ -355,58 +355,55 @@ export class VaModal {
 
     return (
       <Host
-      aria-label={ariaLabel}
-      aria-modal="true"
-      role={ariaRole(status)}
+        aria-label={ariaLabel}
+        aria-modal="true"
+        role={ariaRole(status)}
+        visible={visible}
```
Comment (andresriveratoro) [inline packages/web-components/src/components/va-modal/va-modal.tsx:361]: @bkjohnson This change `visible={visible}` reflects in the DOM allowing the styles to be applied properly when using va-modal inside other web components.

Comment (bkjohnson) [reply packages/web-components/src/components/va-modal/va-modal.tsx:361]: Could we have the `visible` prop have `{reflect: true}` and get the same result?

Comment (andresriveratoro) [reply packages/web-components/src/components/va-modal/va-modal.tsx]: You're correct. Updating to that.

**Review by bkjohnson — APPROVED** [review]
The changes look good to me. I think this should have a `patch` label for publishing though.

**Pull request merged**